src/components/auth/signup/signup-design.html

```diff
@@ -0,0 +1,57 @@
+
+<div class="panel-signup" layout="column" layout-paddings>
+    <md-content class="radius-large md-whiteframe-1dp">
+        <div class="panel-title text-center">
+            <h1 class="no-margin">S3 Portal</h1>
+        </div>
+        
+        <div class="panel-body">
+            <md-subheader class="text-center">Create an account</md-subheader>
+            <form name="@A.1">
+                <md-input-container md-no-float class="md-icon-left md-block">
+                    <md-icon class="material-icons icon-default" ng-class="{'icon-warn:@A.2'}">email</md-icon>
```

Comment:
The `material-icons` of class can be removed because `<md-icon>` will automatically add it.